web/impact/impact/v1/views/organization_list_view.py

```diff
@@ -0,0 +1,59 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+from django.contrib.auth import get_user_model
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from impact.permissions import (
+    V1APIPermissions,
+)
+from impact.models import Organization
+from .organization_detail_view import (
+    organization_is_startup,
+    organization_is_partner,
+    public_inquiry_email,
+)
+
+class OrganizationListView(APIView):
+    permission_classes = (
+        V1APIPermissions,
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.errors = []
+
+    def get(self, request):
+        limit = int(request.GET.get('limit', 10))
+        offset = int(request.GET.get('offset', 0))
+        base_url = request.build_absolute_uri().split("?")[0]
+        result = {
+            "count": Organization.objects.count(),
+            "next": _url(base_url, limit, offset + limit),
+            "previous": _url(base_url, limit, offset - limit),
+            "results": _results(limit, offset),
+            }
+        return Response(result)
+
+    def post(self, request):
+        return Response({"foo": "bar"})                        
+
+def _results(limit, offset):
+    return [serialize_org(org)
+            for org in Organization.objects.all()[offset:offset+limit]]
+                        
+
+def serialize_org(org):
```

Comment:
I also find it weird that this is done in such a different fashion than the serialization in the detail view. Is there a reason for that?